exercises/button_variables_state.md

```diff
@@ -0,0 +1,33 @@
+## Variables
+
+
+Suppose we have a value that represents the number of times a user clicks on a given button. Every time a user clicks on this button then this value increases by 1. We need to refer to this value so we can check it and update it. 
```

Comment:
```suggestion
Suppose we have a value that represents the number of times a user clicks on a given button. Every time a user clicks on this button then this value increases by 1. We will need to keep track of this value so that we can update it. 
```